csharp/test/Apache.Arrow.Tests/ArrowFileReaderTests.cs

```diff
@@ -155,5 +157,19 @@ public async Task TestReadMultipleRecordBatchAsync()
                 ArrowReaderVerifier.CompareBatches(originalBatch1, readBatch3);
             }
         }
+
+        [Fact]
+        public void TestRecordBatchBasics()
+        {
+            RecordBatch recordBatch = TestData.CreateSampleRecordBatch(length: 1);
+            Assert.Throws<ArgumentOutOfRangeException>(() => new RecordBatch(recordBatch.Schema, recordBatch.Arrays, -1));
+
+            var col1 = recordBatch.Column(0);
+            var col2 = recordBatch.Column("list0");
+            ArrowReaderVerifier.CompareArrays(col1, col2);
+
+            recordBatch.Dispose();
```

Comment:
The reason I was calling Dispose() directly is because it seems the Dispose() function was not covered in the unit testing, so by calling it directly, I thought it could increase the code coverage.

Reply:
The `using` statement ensures that `Dispose()` is called, but for increasing the coverage, I think either is fine.